addon/nodes/paragraphWithConfig.ts

```diff
@@ -59,14 +58,11 @@ export const paragraphWithConfig: (
         getAttrs(node: HTMLElement) {
           const nonBlockNode = node.querySelector(BLOCK_SELECTOR);
```

Comment:
We should probably rename this variable to `blockNode`.

Reply:
Hmm, good point.